refinery/core/utils.py

```diff
@@ -21,6 +21,9 @@
 import core
 import data_set_manager
 
+from core.search_indexes import DataSetIndex
+from data_set_manager.search_indexes import NodeIndex
```

Comment:
OK, this is due to mutual imports. Could you add a comment? Also, could you reference this in the circular imports Trello card?